# Mandatory parameters
accel_lim_x: 1.0
accel_lim_y: 1.0
accel_lim_th: 4.0

# Optional parameters
frequency: 20.0
decel_factor: 1.0
